demo/src/routes/+page.svelte

```diff
@@ -13,30 +13,32 @@
 </svelte:head>
 
 <!-- TODO make a list of component and iterate over it when needed -->
-<div class="container-xxl d-flex flex-column align-items-center pt-5">
+<div class="container-xxl d-flex flex-column align-items-center pt-md-5">
```

Comment:
just a little remark here: the blur is cut by the header

Reply:
indeed, though the vertical space gained was worth this small price